src/apps/_experiments/OswAppWeather.cpp

```diff
@@ -379,9 +408,31 @@ void OswAppWeather::printDate() {
 }
 
 bool OswAppWeather::loadData() {
+#ifdef OSW_EMULATOR
+    std::ifstream inFile;
+    inFile.open("file_weather.json"); //open the input file
```

Comment:
where is this file? local or device?

Reply:
I extended the error message below that - but yeah, how to obtain is still a good question